gluten-ut/src/test/scala/org/apache/spark/sql/GlutenJoinSuite.scala

```diff
@@ -17,13 +17,29 @@
 
 package org.apache.spark.sql
 
+import org.apache.spark.sql.internal.SQLConf
+
 class GlutenJoinSuite extends JoinSuite with GlutenSQLTestsTrait {
 
   override def testNameBlackList: Seq[String] = Seq(
```

Comment:
Velox and CH backend share this blacklist, could you please move it into VeloxTestSettings?

Reply:
It depends on whether CH also skips these UTs.